src/managers/types.ts

```diff
@@ -0,0 +1,49 @@
+export enum SafeSettings {
```

Comment:
I can see an import in index.ts for this file:
```
import { ContractCallParams, SafeSettings } from './managers/types'
```

I couldn't find any other ones, so seems not all the types are used.

Reply:
I forgot to remove them. They are not used with the Specific methods for each contract call.